### PR TITLE
Deploy a static site on GitHub Pages.

### PR DESCRIPTION
**Pull request by appleboy**
A GitHub Action to deploy a static site on GitHub Pages.

See https://github.com/appleboy/gh-pages-action

### CODE REVIEW

**Comment by jessicalostinspace** [comment]
This looks like the same/very similar action to "GitHub Actions for deploying to GitHub Pages with Static Site Generators" in the same category. Can you confirm @peaceiris

**Comment by peaceiris** [comment]
Yes. The purpose is similar to https://github.com/peaceiris/actions-gh-pages
On the other hand, peaceiris/actions-gh-pages is implemented with TypeScript.

**Comment by appleboy** [comment]
Some different features:

* deploy to the custom directory in the branch.
* ignore some folder or file list.

**Comment by peaceiris** [comment]
My action can do the first one already and the second is in progress.

@jessicalostinspace 
Why did you ask me about this? According to #345, this repository merges all pull requests regardless that is awesome or not. And maintainers do regular pruning.

File: README.md [hunk 1]
```diff
@@ -307,6 +307,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Action for Hexo](https://github.com/heowc/action-hexo)
 - [Deploy Google Analytics stats to GitHub Pages](https://github.com/cristianpb/analytics-google)
 - [A Jupyter Notebook Blogging Platform Powered by GitHub Actions, Pages and Jekyll](https://github.com/fastai/fastpages)
+- [A GitHub Action to deploy a static site on GitHub Pages (multiple repository)](https://github.com/appleboy/gh-pages-action)
```
Comment (jessicalostinspace) [inline README.md]: @appleboy  please change "A GitHub Action to deploy a static site on GitHub Pages (multiple repository)" to "Deploy A Static Site to GitHub Pages" and add a short description at the end "- Deploy to custom directory and ignore folder/file".

Comment (appleboy) [reply README.md]: @jessicalostinspace Done

**Comment by appleboy** [comment]
@jessicalostinspace Can you merge this PR?

**Comment by jessicalostinspace** [comment]
> @jessicalostinspace Can you merge this PR?

Sure, I want to make sure the sha for the lint fix is in here though and that the action for the repo passes, so would you mind pulling master then pushing up again?

**Comment by appleboy** [comment]
@jessicalostinspace Done

**Comment by appleboy** [comment]
@jessicalostinspace I don't know why CI/CD fail.

**Comment by fabasoad** [comment]
@appleboy try to add `.` (dot) in the end of the sentence.

**Pull request merged**